public/app/features/alerting/unified/api/incidentsApi.ts

```diff
@@ -0,0 +1,26 @@
+import { SupportedPlugin } from '../types/pluginBridges';
+
+import { alertingApi } from './alertingApi';
+
+interface IncidentsPluginConfigDto {
+  isChatOpsInstalled: boolean;
+  isIncidentCreated: boolean;
+}
+
+const getProxyApiUrl = (path: string) =>
+  `/api/plugins/grafana-incident-app/resources/${SupportedPlugin.Incident}${path}`;
+
+export const incidentsApi = alertingApi.injectEndpoints({
+  endpoints: (build) => ({
+    getIncidentsPluginConfig: build.mutation<IncidentsPluginConfigDto, void>({
```

Comment:
(Is this a mutation, really? I think it feels more like a query as we're fetching the data, it just happens to be a post)

Reply:
ops, you're right!